app/src/openxr/cpp/OpenXRInputSource.cpp

```diff
@@ -627,6 +630,17 @@ XrResult OpenXRInputSource::UpdateInteractionProfile()
         }
     }
 
+    // Add haptic devices to controller, if any
+    if (mActiveMapping != nullptr) {
+        uint32_t numHaptics = 0;
+        for (auto& haptic: mActiveMapping->haptics) {
+            if (haptic.hand == OpenXRHandFlags::Both || haptic.hand == mHandeness) {
```

Comment:
Let's get rid of the extra {} for the if as there is just one instruction

Reply:
Fixed.